google-cloud-bigtable/src/main/resources/changestream.json

```diff
@@ -411,13 +411,110 @@
               "low_watermark": "2022-07-01T00:00:00Z",
               "mutations": [
                 {
+                  "set_cell": {
+                    "family_name": "family",
+                    "column_qualifier": "0000000000000000",
+                    "timestamp_micros": 1000,
+                    "value": "dmFsdWUtVkFM"
+                  }
+                }
+              ]
+            }
+          },
+          "error": false
+        }
+      ]
+    },
+    {
+      "description": "1 chunked SetCell([{SC_chunk1}, {SC_chunk2}, {SC_chunk3}]->ChangeStreamMutation{SC})",
```

Comment:
maybe also add the bytestring value to the actual value

Reply:
Done.